Header header
geometry_msgs/Point32 start
geometry_msgs/Point32 end
string color
uint8 action
uint8 ACTION_ADD_MODIFY=0
uint8 ACTION_REMOVE=1
uint8 ACTION_REMOVE_ALL=2